.github/workflows/check-lighthouse-schema.yml

```diff
@@ -23,7 +23,7 @@ jobs:
 
       - name: "Build docker image: api"
         working-directory: infrastructure
-        run: docker compose up api --detach
+        run: docker compose -f docker-compose.yml -f docker-compose.ci.yml up api --detach
```

Comment:
Use an override file for CI services since profiles fails on slightly older versions of docker compose: https://docs.docker.com/compose/extends/#multiple-compose-files